superset-frontend/packages/superset-ui-chart-controls/src/types.ts

```diff
@@ -79,6 +81,7 @@ export interface Dataset {
   description: string | null;
   uid?: string;
   owners?: Owner[];
+  table_name?: string;
```

Comment:
I'm curious, why do we need to append table_name to a column?

Reply:
I'm trying to replace `Datasource` in `src/dashboard/types` with `Dataset`.

```
export type Datasource = Dataset & {
  uid: string;
  column_types: GenericDataType[];
  table_name: string;
};
```

> I'm curious, why do we need to append table_name to a column?

Dataset does not represent a column and can contain an origin table.